.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+
+node_js: "6"
+
+addons:
+  apt:
+    packages:
+      - xvfb
+      - fakeroot
+      - dpkg
+      - libdbus-1-dev
+      - libglib2.0-dev
+      - dbus
+      
+before_install:
+  - cd debian/
+  
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  
```

Comment:
I've added this code so Travis can run tests that require a GUI. At the moment, I can't think of anything but I thought it would be better to have it just in case. @danielravina might think of test with GUI in the future.
We can delete the whole `before_script` section if you want and won't affect Travis (except make it a little faster).